docs/integrations.md

```diff
@@ -0,0 +1,26 @@
+# Integrations
+
+## Weights & Biases
+
+LLM Engine integrates with Weights & Biases to track metrics during fine tuning. To enable:
+
+```python
+from llmengine import FineTune
+
+response = FineTune.create(
+    model="llama-2-7b",
+    training_file="s3://my-bucket/path/to/training-file.csv",
+    validation_file="s3://my-bucket/path/to/validation-file.csv",
+    hyperparameters={"report_to": "wandb"},
+    wandb_config={"api_key":"key", "project":"fine-tune project"}
+)
+```
+
+Configs to specify:
+
+| Field             | Subfield                                                             | Note                                                      |
```

Comment:
Hmm I take that back, maybe a nested bullet point list is fine, e.g. https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/s3/client/create_multipart_upload.html

The fact that the top-level field `wandb_config` is repeated is a bit ugly. Also not ideal that `wandb_config` itself is last (though that's not specific to using the table).